change/@azure-msal-node-2020-08-14-17-01-57-msal-node-client-credential-grant.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "packageName": "@azure/msal-node",
+  "email": "email not defined",
```

Comment:
Need to fix this, otherwise this will show up in the changelog.

Reply:
Added email.